save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt

```diff
@@ -56,7 +57,6 @@ class Save(
     fun performAnalysis(): Reporter {
         logInfo("Welcome to SAVE version $SAVE_VERSION")
         // FixMe: now we work only with the save.toml config and it's hierarchy, but we should work properly here with directories as well
```

Comment:
I think this comment is already outdated and can be removed. Logic with directories is partially implemented and will be improved under #409